<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QWidget" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #00FFFF;
}

</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Капиталлизация процентов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Что за кнопка такая интересная?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate_deposit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_money">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_r">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate_of_payment">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_add">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Частичные снятия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_amount_money">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time_replament">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rate_of_exchange">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>9.8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_money">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: black;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
   </property>
   <property name="text">
    <string> Добавить пополнение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sub_money">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: black;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
   </property>
   <property name="text">
    <string> Добавить снятие</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_time">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rate_of_payment">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #FFC0CB;
	color: black;
	border: 1px solid gray;
	font-size: 16px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #DB7093, stop: 1 #FFC0CB);
}</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>____________________________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>230</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Сумма на вкладе к концу срока:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_money">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>180</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_percent">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Сумма налога:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_tax">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Сумма прибыли:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_amount_profit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_add_money">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Разовое</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в три месяца</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в пол года</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_add_money">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;QLineEdit {
	color: black;
	border: 1px solid gray;
}
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close_add_money">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: red;
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sub_money">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: black;
	border: 1px solid gray;QLineEdit {
	color: black;
	border: 1px solid gray;
}
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close_sub_money">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: red;
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00CED1, stop: 1 #00CDE1);
}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sub_money">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
	border: 1px solid gray;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Разовое</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в три месяца</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в пол года</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
